app/AppKernel.php

```diff
@@ -23,6 +23,7 @@ public function registerBundles()
             new Knp\Bundle\PaginatorBundle\KnpPaginatorBundle(),
             new CodeExplorerBundle\CodeExplorerBundle(),
             new AppBundle\AppBundle(),
+            new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(),
```

Comment:
@dzuelke I tend to agree with you. given that we have a specific installer for the demo in the symfony-installer, we should be loading fixtures there instead (as done here).